modules/transport-netty4/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -17,6 +17,12 @@
  * under the License.
  */
 
+grant {
```

Comment:
Can we grant this only to transport-netty4?

Reply:
I am pretty sure this is handled by the module framework?

Reply:
That is correct. A general grant means all the jars in this plugin.

Reply:
Not quite; it grants it to all JARs in the module and I am asking for this to be limited to transport-netty4 only.

Reply:
Or whatever Netty JAR this is in.

Reply:
That can't work since the doPriv block is in our module code.

Reply:
It could be granted to our jar and the single netty jar explicitly though.

Reply:
Yeah, that should have been an `and`, not an `or`.